### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by GreetyCr**
Completed the tasks on the first microverse project con Week 1 Day 2
Add README.md
Add Linters
Add index.html
Add style.css
Add test.md
Add gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,288 @@
...
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a...
+
```
Comment (ShadyShawkat) [inline README.md:68]: - [ ] Please make sure to customize every part of the readme to better suit your project. Kindly remember that all parts of the readme are required apart from the "FAQ" section.

- [ ] Also please remember to remove the underlining comments under each section header.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @GreetyCr, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (GreetyCr) [reply README.md]: Thank you for the feedback, I did edited but uploaded to the branch the empty template, but I loved the feedback and thanks for helping me become a better developer, it is updated and I will be requesting the second review
Best regards, Greety. @ShadyShawkat

File: README.md [hunk 2]
```diff
@@ -1 +1,173 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
```
Comment (V-Blaze) [inline README.md:8]: - [OPTIONAL] You did a great job implementing the `README` file suggestions by the previous reviewers and also customizing the content. :+1: :100: , I think you can make it more professional by removing the broken logo link. :tada:

**Review by V-Blaze — APPROVED** [review]
Hi @GreetyCr  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights

- Pull request is descriptive 💯
-  Implemented all the changes requested by the previous reviewers 👍🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**